server/src/test/java/io/crate/expression/reference/doc/lucene/LuceneReferenceResolverTest.java

```diff
@@ -47,46 +47,46 @@
 public class LuceneReferenceResolverTest extends CrateDummyClusterServiceUnitTest {
 
     // just return any fieldType to get passt the null check
-    private RelationName name = new RelationName("s", "t");
-    private LuceneReferenceResolver luceneReferenceResolver = new LuceneReferenceResolver(
-        name.indexNameOrAlias(),
+    private static final RelationName RELATION_NAME = new RelationName("s", "t");
+    private static final LuceneReferenceResolver LUCENE_REFERENCE_RESOLVER = new LuceneReferenceResolver(
```

Comment:
Just fyi:

Changing properties in the tests to static also means they can't be GCd after the test-suite lifecycle, leading to higher memory consumption. I'd tend to leave test properties non-final, unless their initialization is so heavy that repeated per-test-case initialization costs actually matter.

Reply:
But gradle spawns new JVM unless you change the `forkEvery`, no?

Reply:
> forkEvery — default: 0 (no maximum)
> This property specifies the maximum number of test classes that Gradle should run on a test process before its disposed of and a fresh one created. This is mainly used as a way to manage leaky tests or frameworks that have static state that can’t be cleared or reset between tests.
> 
> Warning: a low value (other than 0) can severely hurt the performance of the tests


We keep it at the default 0

Reply:
sorry, I had it wrong in my head that 0 -> spawns for each run.